cfme/tests/infrastructure/test_cloud_init_provisioning.py

```diff
@@ -17,15 +19,15 @@
                          required_fields=[['provisioning', 'ci-template'],
                                           ['provisioning', 'ci-username'],
                                           ['provisioning', 'ci-pass'],
-                                          ['provisioning', 'image'],
+                                          ['provisioning', 'ci-image'],
                                           ['provisioning', 'vlan']],
                          scope='module'),
 ]
 
 
 @pytest.fixture(scope="module")
-def setup_ci_template(provisioning, appliance):
-    cloud_init_template_name = provisioning['ci-template']
+def setup_ci_template(provider, appliance):
+    cloud_init_template_name = provider.data.get('provisioning').get('ci-template')
```

Comment:
cloud_init_template_name can't be `None` when passed into `get_template_from_config`, it'll cause things in that method to fail. I think it'd be better to not use `.get()` here and instead hit the `KeyError` if those keys are not found

Reply:
@bsquizz Ok, will do